testgrid/config.yaml

```diff
@@ -2513,6 +2515,11 @@ test_groups:
   num_columns_recent: 20
   alert_stale_results_hours: 24
   num_failures_to_alert: 10
+- name: pull-kubernetes-e2e-kops-aws-coredns
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-kops-aws-coredns
+  num_columns_recent: 20
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 10
```

Comment:
where are these jobs alerting BTW?

i've seen it being used as:
```
lert_options:
      alert_mail_to_addresses: <email>
````

does it make sense to add an email here?